dags/analyzer_helper/discourse/transform_raw_members.py

```
@@ -53,7 +53,7 @@ def transform_member(self, member: dict) -> dict:
             "left_at": None,
             "joined_at": self.converter.from_date_string(member.get("joined_at")),
             "options": {
-                "name": member["name"],
+                "name": member["name"] if member["name"] is not "null" else None,
```

Ruff (0.8.2)

Use `!=` to compare constant literals

Replace `is not` with `!=`

(F632)